packages/server/src/plugin/landingPage/__tests__/getEmbeddedExplorerHTML.test.ts

```diff
@@ -0,0 +1,83 @@
+import { getEmbeddedExplorerHTML } from '../default/index';
```

Comment:
These test files should be moved to `src/__tests__/plugin/landingPage` (and then there's no need to worry about changes to the tsconfig as you mentioned).